www/main/addons/secondary-window/Markdown.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from 'react';
+import ReactMarkdown from 'react-markdown';
+import { ipcRenderer } from 'electron';
+import marked from 'marked';
+import fs from 'fs';
+import path from 'path';
+
+const MarkdownFiles = {
+  help: '../../../../HELP.md',
```

Comment:
we should move these somewhere else. they don't need to be in the root folder. @inderpreetsingh please advice where we should move this.